src/renderer/components/+catalog/catalog.tsx

```diff
@@ -227,7 +227,7 @@ export class Catalog extends React.Component<Props> {
           { title: "Status", className: css.statusCell, sortBy: sortBy.status },
         ]}
         customizeTableRowProps={(item: CatalogEntityItem) => ({
-          disabled: item.isDisabled,
+          disabled: !item.enabled,
```

Comment:
Something is not working (the item is not grayed out/disabled while it is in the deleting phase any more). I don't see what's wrong though

Reply:
Fixed, the problem was I wasn't doing the two writes in an action.